vertica/assets/configuration/spec.yaml

```diff
@@ -94,12 +94,13 @@ files:
             value:
               type: integer
               example: 10
-          - template: instances/tls
-          - name: validate_hostname
-            description: Whether or not to verify the TLS certificate was issued for `server` if `tls_verify` is true.
```

Comment:
Forgot to delete this in the previous PR since the `tls` template already has `tls_validate_hostname`.